src/overlays/actors/ovl_Bg_Ice_Turara/z_bg_ice_turara.h

```diff
@@ -6,9 +6,19 @@
 
 struct BgIceTurara;
 
+typedef void (*BgIceTuraraActionFunc)(struct BgIceTurara*, struct GlobalContext*);
+
+typedef enum {
+    /* 0 */ STALAGMITE,
+    /* 1 */ STALACTITE,
+    /* 2 */ STALACTITE_REGROW
+} BgIceTuraraType;
```

Comment:
Ideally, enum values should have a common prefix (usually based on the actor for actor type enums), so it would be nice if we can get that going here.

We can also figure out later tho, I just wanted to mention it for future cases.

Reply:
I'm happy to change these now, although from a purely functional point of view I doubt they'll get confused with anything else